public/src/main/java/com/revenuecat/purchases/EntitlementInfo.kt

```diff
@@ -49,7 +49,8 @@ data class EntitlementInfo constructor(
     @Deprecated(
         "Use rawData instead",
         replaceWith = ReplaceWith("rawData")
-    ) val jsonObject: JSONObject
+    ) val jsonObject: JSONObject,
+    val verification: VerificationResult
```

Comment:
This and the change in `EntitlementInfos` are technically breaking changes, that's part of what I wanted to discuss over slack. In this case, I could default to `NOT_VERIFIED` but I don't think that's ideal.